### ISSUE TITLE
Possibility to provide your own config directory

### ISSUE THREAD

**Issue by jkaan** [sharkdp/bat#145]
Hello again!

I am wondering if it's already possible to supply a custom configuration directory. I have my dotfiles in a folder in my home directory (~/.dotfiles) and it would be awesome if I could point bat directly at that folder to generates its caches of themes and syntaxes.

**Comment by sharkdp** [comment]
Yes, this will be possible in the next version (or with the current `master`). `bat cache` now offers a few more options:

```
▶ bat cache -h
bat-cache 
Modify the syntax-definition and theme cache

USAGE:
    bat cache <--init|--clear|--config-dir>

OPTIONS:
    -i, --init
            Initialize the syntax/theme cache

    -c, --clear
            Reset the cache

    -d, --config-dir
            Show the configuration directory

        --source <dir>
            Use a different source for loading syntaxes and themes from

        --target <dir>
            Use a different source to store the cached syntax and theme set

    -h, --help
            Prints help information
```

**Comment by jkaan** [comment]
@sharkdp Nice! Well thanks for answering this question so fast again :).

**Comment by sharkdp** [comment]
Fixed in v0.4
